docs/rfc/active/business_glossary/README.md

```diff
@@ -0,0 +1,224 @@
+|--|--|--|--|--|--|--|--|
+|urn:li:businessTerm:(instrument.cashInstrument) | instrument.cashInstrument| time point including a date and a time, optionally including a time zone offset| Foundation | abc@domain.com | fibo | fibo-fbc-fi-fi:CashInstrument|https://spec.edmcouncil.org/fibo/ontology/FBC/FinancialInstruments/FinancialInstruments/CashInstrument |
+|urn:li:businessTerm:(common.dateTime) | common.dateTime| a financial instrument whose value is determined by the market and that is readily transferable (highly liquid)| Finance | xyz@domain.com | fibo | fibo-fnd-dt-fd:DateTime|https://spec.edmcouncil.org/fibo/ontology/FND/DatesAndTimes/FinancialDates/DateTime |
+|urn:li:businessTerm:(market.bidSize) | market.bidSize| The bid size represents the quantity of a security that investors are willing to purchase at a specified bid price| Trading | xyz@domain.com | - | - |- | - |
+|--|--|--|--|--|--|--|--|
+| | | | | | | | |
+
+### Business Term  & Dataset - Relationship
+
+| Attribute Name| Data Type| Nullable?| **Business Term**| Description|
+|--|--|--|--|--|
+| recordId| int| N| | |n the case of FX QuoteData the RecordId is equal to the UIC from SymbolsBase|
+| arrivalTime| TimestampTicks| N| | Time the price book was received by the TickCollector. 100s of Nanoseconds since 1st January 1970 (ticks)|
+| bid1Price| com.xxxx.yyy.schema.common.Price| N| **common.monetoryAmount**|The bid price with rank 1/29.|
+| bid1Size| int| N| market.bidSize| The amount the bid price with rank 5/29 is good for.|
+|--|--|--|--|--|--|--|
+| | | | | | | |
+
+### Stiching Together
+
+
+Business Glossary will be a first class entity where one can define the business terms and this will be similar to entities like Dataset, CorporateUser etc.
+Then extending the Dataset's aspect SchemaMetadata, that will be extending the SchemaField model to capture urn of the business term (optional).
+
+![high level design](business_glossary_rel.png)
+
+The above diagram illustrates how business glossaries will be connected to other entities. Business terms are modelled as BusinessTerm entities and example business terms are `Term-1`, `Term-2`, .. `Term-n`. In the above diagram from the dataset (`DS-1`) element `e11` is linked to business term `Term-2` and `e12` is linked to `Term-1`. At the same time from Dataset (`DS-2`) element `e24` linked the business term `Term-2`, `e22` with `Term-3` and `e24` with `Term-4`.
+
+## Metadata Model Enhancements 
+
+There will be 1 top level GMA [entities](../../../what/entity.md) in the design: businessTerm (Business Glossary).
+It's important to make businessTerm as a top level entity because it can exist without a Dataset and can be defined independently by the business team.
+
+### URN Representation
+We'll define a [URNs](../../../what/urn.md): `BusinessTermUrn`.
+These URNs should allow for unique identification of business term.  
+
+A business term  URN will look like below:
+```
+urn:li:businessTerm:(<<namespace>>,<<name>>)
```

Comment:
Have you thought about the need for renaming the business glossary ? Say for instance, we decide to rename the term `market.bidSize` to `market.bidVolume`. 

An alternate way of URN modelling is making the URN opaque `urn:li:businessTerm:<id_int>`. The rest of the metadata as in the namespace and name can go into one of the aspect definitions.

Reply:
I propose not allow renaming of the business term, at the max user should mark the existing one for deprecation/delete and define a new one